nmosquery/common/query.py

```diff
@@ -221,7 +221,7 @@ def do_sync(self, ws, socket):
 
     def do_sup(self, path, pre_obj, post_obj):
         self.logger.writeDebug('do_sup {} {}'.format(self.api_version, path))
-        if cmp(post_obj, pre_obj) == 0:
+        if post_obj == pre_obj:
```

Comment:
As discussed, this one will need to change to deal with the potential for dicts with different ordering.